modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/creationActions.spec.ts

```diff
@@ -608,3 +608,89 @@ test(
 		});
 	}
 );
+
+test(
+	'Deactivate and activate a creation action',
+	{tag: '@LPD-39965'},
+	async ({actionsPage, dataSetManagerApiHelpers, page}) => {
+		const icon: string = 'arrow-right-full';
+		const label: string = getRandomString();
+		const type: ECreationActionTarget = ECreationActionTarget.LINK;
+
+		await test.step('Create an item action with API', async () => {
+			await dataSetManagerApiHelpers.createDataSetCreationAction({
+				dataSetERC,
+				icon,
+				label_i18n: {en_US: label},
+				target: ECreationActionTarget.LINK,
+			});
+		});
+
+		await test.step('Go to creation actions tab', async () => {
+			await actionsPage.gotoCreationActionsTab({dataSetLabel});
+		});
+
+		let actionRow: Locator;
+
+		await test.step('New creation action is displayed on the table and is "Active" by default', async () => {
+			await expect(actionsPage.creationActionsTab).toBeInViewport();
+
+			actionRow = await getRowByText({
+				page,
+				table: actionsPage.creationActionsTable,
+				text: label,
+			});
+
+			await expect(actionRow.getByRole('cell')).toContainText([
+				icon,
+				label,
+				type,
+				'Active',
+			]);
+
+			await expect(actionsPage.activeToggle.last()).toBeVisible();
```

Comment:
is it fine to check the last occurrence? do we know for sure there'll be only one action?